### PR TITLE
Moebius R&D Redux & AK resprite

### PR DESCRIPTION
**Pull request by shazbot194**
## About The Pull Request

This Creates a new standard for Moebius sprites, using a white and black color scheme with purple stripes. It also redoes almost all of the R&D sprites, including in_hands. 

image

The colors

image

Since it is very similar, this pull also redoes AK sprites. I did kind of drop the ball on the first pass, doubly so for the in_hands. They should now be the same as everything else. From feedback and executive 'asking', I also removed 2 of the 3 FS AKs, leaving the Vepr.

You can also fold the Krinkov and Venger's stocks with ctrl+shift+click, and it plays a sound.

image

## Why It's Good For The Game

Creates a stronger style for one of our main factions, while making one weapon less of eyesores.

## Changelog
:cl: Shazbot194
tweak: New Moebius R&D pallate.
del: Removed Wasp and Brushmaster
tweak: All AKs have had their sprites updated
tweak: Ctrl+Shft+click now folds AK stocks.
/:cl:

### CODE REVIEW

**Comment by zombiemimic** [comment]
I feel like this should be two different PRs

**Comment by shazbot194** [comment]
Not disagreeing but firefox said put both in one.

File: code/modules/research/designs/weapon.dm [hunk 1]
```diff
@@ -23,9 +23,9 @@
 	sort_string = "TAAAC"
 
 /datum/design/research/item/weapon/c20r
-	name = "C20R-prototype"
-	desc = "The C-20r is a lightweight and rapid-firing SMG. Uses 10mm rounds."
-	build_path = /obj/item/weapon/gun/projectile/automatic/c20r
+	name = "C20M-prototype"
+	desc = "The C-20M is a lightweight and rapid-firing SMG. Uses .35rin rounds."
```
Comment (Firefox13) [inline code/modules/research/designs/weapon.dm:27]:
```suggestion
	desc = "The C-20M is a lightweight and rapid-firing SMG. Uses .35 Auto rounds."
```

**Comment by L74814** [comment]
epic

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**